js/main.js

```diff
@@ -20,7 +32,69 @@ function submitHandler(event) {
   data.nextEntryId++;
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  $ul.prepend(renderEntry(newEntry));
 }
 
-$photoUrl.addEventListener('input', inputHandler);
-$form.addEventListener('submit', submitHandler);
+function renderEntry(entry) {
+  var $li = document.createElement('li');
+  var $divRow = document.createElement('div');
+  $divRow.setAttribute('class', 'row');
+  var $divColumnHalfImg = document.createElement('div');
+  $divColumnHalfImg.setAttribute('class', 'column-half');
+  var $img = document.createElement('img');
+  $img.setAttribute('src', entry.image);
+  $img.setAttribute('alt', entry.title);
+  var $divColumnHalfText = document.createElement('div');
+  $divColumnHalfText.setAttribute('class', 'column-half');
+  var $h2 = document.createElement('h2');
+  $h2.textContent = entry.title;
+  var $p = document.createElement('p');
+  $p.textContent = entry.notes;
+  $li.appendChild($divRow);
+  $divRow.appendChild($divColumnHalfImg);
+  $divColumnHalfImg.appendChild($img);
+  $divRow.appendChild($divColumnHalfText);
+  $divColumnHalfText.appendChild($h2);
+  $divColumnHalfText.appendChild($p);
+  return $li;
+}
+
+function loadHandler(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    var newTree = renderEntry(data.entries[i]);
+    $ul.appendChild(newTree);
+  }
+  for (var x = 0; x < $allView.length; x++) {
+    if ($allView[x].getAttribute('data-view') !== data.view) {
+      $allView[x].className = 'view hidden';
+    } else {
+      $allView[x].className = 'view';
+    }
+  }
+  if (data.entries.length !== 0) {
+    $noEntryP.className = 'no-entry-p hidden';
+  }
```

Comment:
This looks like a lot of the same code as `viewHandler`. You might want to rewrite that function to use a parameter rather than the data straight from `event.target`. That way, you can call that function any time you like and just pass in the view you want to show as an argument wherever you call the function